tutorial/markdown/python/bedrock-serverless-pdf-chat/bedrock-serverless-pdf-chat.md

```diff
@@ -0,0 +1,660 @@
+---
+# frontmatter
+path: "/tutorial-bedrock-serverless-pdf-chat"
+# title and description do not need to be added to markdown, start with H2 (##)
+title: Build PDF Chat App With Couchbase, AWS Bedrock using serverless arch
```

Comment:
arch -> architecture

Reply:
Do you have a title in mind? I am confused like I can remove words like `using serverless arch`  to make it fit but wanted to showcase power of serverless here.

The issue is that title has a character limit of 72